### ISSUE TITLE
12.9.0 disables MBP caps-lock LED

### ISSUE THREAD

**Issue by peterpqa** [pqrs-org/Karabiner-Elements#2124]
Karabiner-Elements 12.9.0 completely disabled my caps-lock LED (2015 MacBook Pro, macOS 10.15.2). I tried simply uninstalling 12.9.0 and reinstalling 12.8.0, but my LED did not come back. I had to find and delete all files and folders with a name containing "karabiner" or "pqrs", restart, then install 12.8.0. After that, my LED came back. (There was probably only one file I needed to delete after uninstalling, but I didn't know which one, so I deleted them all.)

**Comment by rzuritamilla** [comment]
I have the same issue on my MBP 16" (10.15.2). I did not uninstall K-E because I am hoping for a quick fix or a new version :-) Thanks!

**Comment by peterpqa** [comment]
I will not be updating from 12.8. The only thing I need K-E for is [my FN+ jkl to 123 .json file](https://github.com/pqrs-org/Karabiner-Elements/issues/2125). It works just fine and my caps-lock LED works just fine. No more updates for me...

**Comment by elfive** [comment]
device:MBP 2015 15'' with TouchBar
system: MacOS 10.15.2
karabiner version:12.9.0
symptoms: LED working in system login view, after login, led remains last status in system login view.
maybe conflicting with USB Overdrive or logitech game software.

**Comment by jec0047** [comment]
On my iMac Pro, KE 12.9.0 not only breaks my caps-lock LED, it breaks caps-lock altogether.

**Comment by rzuritamilla** [comment]
@jec0047: +1
First I thought that it was the LED only but a couple of weeks ago I realized that my caps-lock key does not work at all...

**Comment by rzuritamilla** [comment]
Hey, I just found a fix for this issue. Please check [this](https://www.40tech.com/2019/07/25/make-your-caps-lock-key-do-double-duty-as-a-modifier-key-and-caps-lock/). I copied the complete "to_if_alone" section to my JSON file. Then I restarted KE and voilá.: All OK (typed with my restored caps-lock key, with functional LED :-))

**Comment by jec0047** [comment]
> Hey, I just found a fix for this issue...

Unfortunately this did not fix my problem. As soon as I run KE, _without any modifications installed_, caps-lock stops working.

**Comment by chinnnki** [comment]
> Hey, I just found a fix for this issue. Please check [this](https://www.40tech.com/2019/07/25/make-your-caps-lock-key-do-double-duty-as-a-modifier-key-and-caps-lock/). I copied the complete "to_if_alone" section to my JSON file. Then I restarted KE and voilá.: All OK (typed with my restored caps-lock key, with functional LED :-))

I’m sorry it didn't fix the issue for me. Rolled back to 12.8 and everything’s fine again.

**Comment by jec0047** [comment]
This is still a problem as of 5/23/2020.

**Comment by larsof54** [comment]
I have it working on my MacBook Pro 2015, using Mojave 10.14.6, KE 9.0.5, KE 12.9.0. I loaded in a json file that I got from  somewhere out there and it started working. I have BTT, KE and KM running concurrently along with a few other apps. Here it is if anyone wants to try it...
[working tap to capslock hold for hyper.zip](https://github.com/pqrs-org/Karabiner-Elements/files/4672687/working.tap.to.capslock.hold.for.hyper.zip)
